client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { noop } from 'lodash';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'EditGravatar', function() {
```

Comment:
We chatted over this. Since I'm using `mockery` only for mocking `event`, `matches-selector` and `lib/oauth-token`, and `shallow` for the tests, we're all set.